facebook_connector.py

```diff
@@ -45,8 +52,23 @@ def resolve_user_id(user_id):
     Returns:
         The name of the user if it is able to find it, otherwise the input
     """
-    # TODO (neitsch): This method is not implemented yet.  Use the user_id
-    # (format: 12345678@facebook.com) to get the actual name of the user
+    fb_id_pattern_match = _FB_ID_PATTERN.match(user_id)
+    if fb_id_pattern_match:
+        fb_numeric_id = fb_id_pattern_match.group(1)
+        try:
+            if fb_numeric_id not in _mapped_fb_ids:
+                fb_user_page = urlopen("https://www.facebook.com/{}".format(fb_numeric_id))
```

Comment:
To verify that it resolves IDs I printed them every time a new one is resolved. Definitely works.